3 - PB/Documentazione esterna/Manuale Utente/Manuale Utente.typ

```diff
@@ -151,6 +151,52 @@ Premere quindi il pulsante *Submit* per procedere alla visualizzazione dell'ambi
 
 Una volta che l'ambiente è stato correttamente configurato è possibile cominciare a lavorare con il piano 3D che si presenta come mostrato in in @fine_configurazione_iniziale (in questo caso i piani rappresentati sono entrambi vuoti).
 
+== Movimento nell'ambiente tridimensionale
+Il sistema permette tre diversi tipi di movimento di camera all'interno dell'ambiente: rotazione del piano, zoom-in e zoom-out e lo spostamento della camera sui due assi (panning).
+
+=== Rotazione della piano
+
+#figure(
+    grid(
+        columns: 3,
+        rows:    (auto, auto),
+        [ #image("./imgs/rotazione_1.png", width: 80%)],
+        [ #image("./imgs/rotazione_2.png", width: 75%)],
+        [ #image("./imgs/mouse_tasto_sinistro.png", width: 50%)],
+    ),caption: [Rotazione del piano rispetto la camera],
+) <camera_rotazione>
+
+Premendo il tasto sinistro e spostando il mouse verso destra ruoteremo il piano rispetto alla camera in senso antiorario, muovendolo invece verso sinistra verrà ruotato in senso orario, come mostrato in @camera_rotazione. Nello stesso modo, spostando il mouse verso il basso possiamo cambiare l'angolazione del piano.
+
+=== Zoom-in, Zoom-out
+
+#figure(
+    grid(
+        columns: 3,
+        rows:    (auto, auto),
+        [ #image("./imgs/zoom-in.png", width: 80%)],
+        [ #image("./imgs/zoom-out.png", width: 75%)],
+        [ #image("./imgs/mouse_rotella.png", width: 30%)],
+    ),caption: [Zoom-in e zoom-out],
+) <camera_zoom>
+
+Ruotando la rotella del mouse in avanti la camera si avvicinerà all'oggetto che desideriamo ingrandire (zoom-in), girandola al contrario la camera si allontanerà dall'oggetto (zoom-out) come mostrato in @camera_zoom.
```

Comment:
Modificherei il paragrafo in

Ruotando la rotella del mouse in avanti è possibile avvicinarsi all'oggetto desiderato (zoom-in), mentre ci si può allontanare (zoom-out) ruotandola nel senso opposto, come mostrato in `@camera_zoom`.